site/docs/components/system-status/index.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: System status
+data:
+  description: "System status communicates essential platform-wide information or alerts, such as system maintenance or outages. These messages are designed to be quickly noticed and prompt users to take action if needed. System status offers 4 severity levels, each with a dedicated icon and color; `info`, `success`, `warning` and `error`. "
+  sourceCodeUrl: "https://github.com/jpmorganchase/salt-ds/blob/main/packages/lab/src/system-status"
+  package:
+    name: "@salt-ds/lab"
```

Comment:
Not sure why 🚧 not appearing in side bar

Reply:
The site is not updated. Have asked Josh to deploy the latest build.

Reply:
I deployed it yesterday after this comment

Reply:
its visible on site now :)